release.sh

```diff
@@ -38,39 +38,10 @@ set -ex
 ROOTDIR="$(dirname "${BASH_SOURCE[0]}")"
 source "${ROOTDIR}/vendor/github.hpe.com/hpe/hpc-shastarelm-release/lib/release.sh"
 
-# Set to "unstable" for artifacts from main branch, "stable" for artifacts from
```

Comment:
Ooops. Bad cherry-pick here. Fixing.

Reply:
I re-did this cherry-pick and made sure to include the call to the new `render_templates.sh` script.